cmd/hypervisor/commands/root.go

```diff
@@ -2,7 +2,7 @@ package commands
 
 import (
 	"fmt"
-	"net"
+	"github.com/SkycoinProject/dmsg/disc"
```

Comment:
Could you please format imports splitting them to 3 blocks?
- standard library imports
- other repositories
- subfolders of `github.com/SkycoinProject/skywire-mainnet`

Reply:
Done